cfn-resources/stream-connection/cmd/resource/mappings_test.go

```diff
@@ -263,6 +263,38 @@ func TestGetStreamConnectionClusterTypeModel(t *testing.T) {
 	})
 }
 
+func TestGetStreamConnectionSampleTypeModel(t *testing.T) {
+	streamsConnSample := &admin.StreamsConnection{
+		Name: ptr.String("sample_stream_solar"),
+		Type: ptr.String("Sample"),
+	}
+	t.Run("With Nil Current Model", func(t *testing.T) {
```

Comment:
knitpit: it's normally recommended to use the table-driven approach

Reply:
Done in 2ce1217 ✅ 
@lantoli 
Can you double-check that I did it correctly?